openedx/features/course_experience/utils.py

```diff
@@ -15,7 +15,7 @@
 
 
 @request_cached()
-def get_course_outline_block_tree(request, course_id, user=None, allow_start_dates_in_future=False):  # lint-amnesty, pylint: disable=too-many-statements
+def get_course_outline_block_tree_accessed(request, course_id, user=None, allow_start_dates_in_future=False):  # lint-amnesty, pylint: disable=too-many-statements
```

Comment:
The location needs to be at openedx/features/enterprise_support/utils.py

Reply:
i made the changes kindly check